src/selectors/index.test.js

```diff
@@ -0,0 +1,80 @@
+import { NETWORK_KINDS } from 'constants/network';
+import {
+  getNetwork,
+  getBeliefs,
+  getSelectedNodes,
+  getSubnetworks,
+  getNetworkKind,
+  getPanelVisibility,
+  getLinkages,
+} from './index';
+
+
+describe('Main Selectors', () => {
+  describe('getNetwork selector', () => {
+    const network = {};
+    const store = { network };
+    it('gets network', () => {
+      expect(getNetwork(store)).toEqual(network);
+    });
+  });
+
+  describe('getBeliefs selector', () => {
+    const beliefs = [42];
+    const network = { beliefs };
+    const store = { network };
+    it('gets Beliefs', () => {
+      expect(getBeliefs(store)).toEqual(beliefs);
+    });
+  });
+
+  describe('getSelectedNodes selector', () => {
+    const selectedNodes = [1, 2, 3];
+    const network = { selectedNodes };
+    const store = { network };
+    it('gets selected nodes', () => {
+      expect(getSelectedNodes(store)).toEqual(selectedNodes);
+    });
+  });
+
+  describe('getSubnetworks selector', () => {
+    const subnetworks = [1, 2, 3];
+    const network = { subnetworks };
+    const store = { network };
+    it('gets subnetworks', () => {
+      expect(getSubnetworks(store)).toEqual(subnetworks);
+    });
```

Comment:
This selector has two behaviors:
- When network has subnetworks
getSubnetworks = state => **state.network.subnetworks** || []
- When network has no subnetworks
getSubnetworks = state => state.network.subnetworks || **[]**

So, I would update this test to:
```js
describe('getSubnetworks selector', () => {
  desribe('When network has subnetworks', () => {
    const subnetworks = [1, 2, 3];
    const network = { subnetworks };
    const store = { network };

    it('gets subnetworks', () => {
      expect(getSubnetworks(store)).toEqual(subnetworks);
    });
  });

  desribe('When network has no subnetworks', () => {
    const network = { };
    const store = { network };
    
    it('gets subnetworks', () => {
      expect(getSubnetworks(store)).toEqual(subnetworks);
    });
  });
});
```